packages/hero/src/scss/_component.scss

```diff
@@ -124,4 +128,15 @@
    
   }
 
+  &--large {
+
+    & .uq-hero__content {
```

Comment:
Not a fan of the specificity here. But it would require rebuilding the current Hero.